src/ThirdParty/utils.spec.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  ThirdPartyFragment,
+  ThirdPartyMetadataType,
+} from '../ethereum/api/fragments'
+import { toThirdParty } from './utils'
+import { ThirdParty } from './ThirdParty.types'
+
+describe('toThirdParty', () => {
+  describe('when a fragment is supplied', () => {
+    let fragment: ThirdPartyFragment
+
+    beforeEach(() => {
+      fragment = {
+        id: 'some:id',
+        managers: ['0x1', '0x2'],
+        maxItems: 1,
+        totalItems: 1,
+        metadata: {
+          type: ThirdPartyMetadataType.THIRD_PARTY_V1,
+          thirdParty: {
+            name: 'a name',
+            description: 'some description',
+          },
+        },
+      }
+    })
+
+    it('should take a third party fragment and parse it to conform to the ThirdParty type', async () => {
```

Comment:
We can remove the async here, as we're not using the await keyword:
```suggestion
    it('should take a third party fragment and parse it to conform to the ThirdParty type', () => {
```